responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html

```diff
@@ -285,41 +285,49 @@
               <div class="col-12">
                 <div class="review-panel data-panel">
                   <label class="tab-subtitle">Needs Assessment</label>
```

Comment:
Same - bind to the array instead of if/then/else

Reply:
same situation, display text is different from object name.  if you really wanted I suppose I could store all the value/name pairs in the ts file and read each one but that seems like another way of doing essentially the same thing